src/locale/misc/formDialog/en.yml

```diff
@@ -1,26 +1,27 @@
-createNew: 
-    heading: Create new {resource}
-    campaign:
-        name: Name
-        description: Description
-        visibility:
-            heading: Visibility
-            private: Private
-            public: Public
-        status:
-            heading: Status
-            published: Published
-            draft: Draft
-    event:
-        campaign: Campaign
-        title: Title
-        link: Link
-        start: Start Time
-        end: End Time
-        activity: Activity
-        place: Place
-        info: Information on signup
-        minNo: Mininum participants
+create: Create new {resource} 
+edit: Edit {resource} 
+campaign:
```

Comment:
If you want to, you could create a separate `campaign` folder and put the contents below in an `en.yml` in that folder (and do the same to `event`), to avoid having all the forms in the same file.